apps/src/templates/teacherDashboard/SectionsPage.jsx

```diff
@@ -124,6 +130,14 @@ class SectionsPage extends Component {
             color={ProgressButton.ButtonColor.gray}
           />
         }
+        {sectionsLoaded && showCleverClassroom &&
+        <ProgressButton
+          text={i18n.importFromCleverClassroom()}
+          style={styles.button}
+          onClick={this.handleImportOpen}
+          color={ProgressButton.ButtonColor.gray}
+        />
```

Comment:
ProgressButton block should be indented I think (tho it's quite possible we're not consistent about this)